R/adjustcarryforward.R

```diff
@@ -226,7 +233,7 @@ adjustcarryforward <- function(subjid,
       agedays < 0, 'Missing', 'No Change'
   )),
   levels = exclude.levels,
-  ordered = T)]
+  ordered = T)] # why is this ordered??
```

Comment:
Not sure why this is ordered! But it looks like this was carried forward (ugh, sorry) from the pattern established in `growth.R`, where it is also ordered in several places.

Reply:
Just in case this is something you remember, @grundmeier or @dcam2015, do you recall how/whether the factor level ordering was used? After a quick skim, I'm not necessarily seeing it, but of course I'm probably missing something!

Reply:
The ordering did not end up being important, as you noted. Since the "cleaning" process is ordered into a series of numbered steps, my theory originally was that individuals using the code might have wanted to be able to use > or < comparisons to conveniently filter results based on where in the processing a particular measurement was excluded. But, it isn't important.

Reply:
Thanks for confirming!